cmd/maya-exporter/app/collector/jiva.go

```diff
@@ -0,0 +1,111 @@
+// Package collector is used to collect metrics by implementing
+// prometheus.Collector interface. See function level comments
+// for more details.
+package collector
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/openebs/maya/types/v1"
+	"github.com/pkg/errors"
+
+	"github.com/golang/glog"
+)
+
+// jiva implements the Volume interface. It exposes
+// the metrics of a OpenEBS (Jiva) volume.
+type jiva struct {
+	url   string
+	stats stats
+}
+
+func Jiva(url *url.URL) *jiva {
```

Comment:
Need to provide comments for all public functions, methods, variables, constants.